composer.json

```
@@ -4,26 +4,26 @@
     "type": "library",
     "license": "GPL-2.0-only",
     "require": {
-        "php": ">=7.2.0",
+        "php": ">=8.0.0",
         "ext-json": "*",
         "ext-openssl": "*",
         "codercat/jwk-to-pem": "^1.0",
-        "guzzlehttp/guzzle": "^6.5 || ^7.0",
-        "lcobucci/jwt": "^3.4 || ^4.1.5",
-        "league/oauth2-server": "^8.2",
-        "nesbot/carbon": "^2.43",
-        "nyholm/psr7": "^1.3",
+        "guzzlehttp/guzzle": "^7.7.0",
+        "lcobucci/jwt": "^4.3.0",
+        "league/oauth2-server": "^8.5",
+        "nesbot/carbon": "^2.71",
+        "nyholm/psr7": "^1.8",
         "php-http/message-factory": "^1.1",
         "phpseclib/phpseclib": "^3.0.34",
-        "psr/cache": "^1.0 || ^2.0 || ^3.0",
+        "psr/cache": "^2.0 || ^3.0",
         "psr/http-message": "^1.0",
         "psr/http-server-handler": "^1.0",
-        "psr/log": "^1.0 || ^2.0 || ^3.0",
-        "ramsey/uuid": "^3.9 || ^4"
+        "psr/log": "^2.0 || ^3.0",
```

Let's leave the older psr/log here for migration reasons as well.

```suggestion
        "psr/log": "^1.0 || ^2.0 || ^3.0",
```

---

@wazelin no problem.

if we already bump versions, I'll revise this one, cause PHP 8.0 is also EOL (by now).